src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/add_order_discount_modal.html.twig

```diff
@@ -66,7 +66,7 @@
 
                 <div class="input-group-append">
                   <div class="input-group-text" id="add_order_cart_rule_value_unit" data-currency-symbol="{{ orderCurrency.symbol }}">%</div>
-                  {{ form_widget(addOrderCartRuleForm.value, {'type':'number'}) }}
+                  {{ form_widget(addOrderCartRuleForm.value, { 'type':'number', 'attr': { 'step': 1, 'class': 'input-group-text' }}) }}
```

Comment:
These arguments should be integrated in the form type rather than in the twig Especially for the type number

Reply:
Done